### ISSUE TITLE
[Bug]: Github Action refuses to download 0.4.4

### ISSUE THREAD

**Issue by tclain-pillar** [jetpack-io/devbox#790]
**Current Behavior (bug)**

Recently our CI (github action) has started failing with this error:

```
Run jetpack-io/devbox-install-action@v0.2.0
Run if [[ -n $DEVBOX_USE_VERSION ]]; then
Run actions/cache@v3
Cache not found for input keys: Linux-devbox-0.2.2
Run DEVBOX_USE_VERSION="0.2.2"
Devbox 📦 by jetpack.io
  Instant and predictable development environments and containers.

  This script downloads and installs the latest devbox binary.

Confirm Installation Details
  Location:     /usr/local/bin/devbox
  Download URL: https://releases.jetpack.io/devbox


Downloading and Installing
→ Downloading devbox binary...
✓ Downloading devbox binary... [DONE]
→ Installing in /usr/local/bin/devbox (requires sudo)...
✓ Installing in /usr/local/bin/devbox... [DONE]
✓ Successfully installed devbox 🚀

Next Steps
  1. Learn how to use devbox
     Run devbox help or read the docs at https://github.com/jetpack-io/devbox
  2. Get help and give feedback
     Join our community at https://discord.gg/jetpack-io
→ Downloading version 0.4.4...
✓ Downloading version 0.4.4... [DONE]
→ Verifying checksum...
✓ Verifying checksum... [DONE]
→ Unpacking binary...
✓ Unpacking binary... [DONE]

ERROR: mismatch devbox version downloaded. Expected 0.2.2, got 0.4.4.`

```

Our Github file looks like (redacted)

```yaml
name: CI-integration

on: push

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - name: Install devbox
        uses: jetpack-io/devbox-install-action@v0.2.0
        with:
          devbox-version: 0.4.4
```

**Expected Behavior (fix)**
- 0.4.4 version should be installed

**Additional context**
Please include the output of `devbox version -v` and
a copy of your `devbox.json` file.

```json
{
  "packages": ["nodejs", "nodePackages.pnpm", "postgresql"],
  "shell": {
    "init_hook": null
  },
  "nixpkgs": {
    "commit": "f80ac848e3d6f0c12c52758c0f25c10c97ca3b62"
  },
  "scripts": {
    "test-integration": "echo \"World\"",
    "echo_twice": ["echo \"World\"", "echo \"Again\""]
  }
}

```

**Comment by loreto** [comment]
@tclain-pillar The output you pasted shows this line:
> Run DEVBOX_USE_VERSION="0.2.2"

Is that an environment variable you are setting somewhere else in your github actions by any chance?

**Comment by tclain-pillar** [comment]
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/99363042/226455251-c99a3a86-1071-4fec-a697-9de994427786.png">

**Comment by LucilleH** [comment]
Hi @tclain-pillar, I just published `v0.3.0`. Do you mind trying the github action with:
```
uses: jetpack-io/devbox-install-action@v0.3.0
```

**Comment by tclain-pillar** [comment]
@LucilleH that seems to have done the trick. thanks!
